### DISCUSSION TITLE
Die source 6000 für 4060 ist crap

### DISCUSSION THREAD

**Discussion by SaMMy-lacht** [Freetz-NG/freetz-ng#811] [Allgemein]
Das Verwenden der source von der 6000 auf den 4060 ist crap wie du es so schön immer sagt 
gibt Warnungen aus 
`depmod: WARNING: build/modified/filesystem/lib/modules/4.4.60/kernel/drivers/dsld/kdsldmod.ko needs unknown symbol generic_ct_destroy`
`depmod: WARNING: build/modified/filesystem/lib/modules/4.4.60/kernel/drivers/dsld/kdsldmod.ko needs unknown symbol rt_dst_alloc`
`depmod: WARNING: build/modified/filesystem/lib/modules/4.4.60/kernel/drivers/dsld/kdsldmod.ko needs unknown symbol generic_ct_create`

mit der Verwendung der richtigen source also die von der 4060 ist dieses aber nicht mehr 

Test Kernel Datein 
kernel_4.4.60-4060_07.30-ADv1.patch.xz 4,8 MB (4.756.012 Bytes)
kernel_4.4.60-6000_07.30-ADv1.patch.xz 4,7 MB (4.737.400 Bytes)
kernel_4.4.60-6000_07.27-ADv1.patch.xz 4,7 MB (4.737.400 Bytes)

**Comment by fda77** [comment]
Du hast nicht so ganz verstanden was ich so bezeichne. Jedenfalls nicht wenn man (offenbar falsche) Sourcen von einem anderen Gerät nimmt...
Seh ich es richtig dass die 4060 sourcen in freetz noch fehlen?

```
kernel_4.4.60-6000_07.30-ADv1.patch.xz 4,7 MB (4.737.400 Bytes)
kernel_4.4.60-6000_07.27-ADv1.patch.xz 4,7 MB (4.737.400 Bytes)
```

damit willst du sagen dass es zwischen .27 und .30 keine Änderungen gab?

**Reply by fda77** [reply]
die 7510 nutzt auch 6000 src, daran hab ich ne notiz "geht nicht", ka warum das so war

**Reply by SaMMy-lacht** [reply]
Das ist doch auch total logisch, dass es nicht geht FW 7.5X Kernel 4.4.271 (was bei der 7510 ja zutrifft) 4060 FW 7.2X Kernel 4.4.60

Die 6000 source ist für die 7510 auch mal gar nicht geeignet. Es wird auch keine source für die 7510 fw 7.2x geben.

**Comment by fda77** [comment]
Ich hab mir FREETZ_AVM_SOURCE_FOR_SYSTEM_TYPE_QCAARMv8 in https://github.com/Freetz-NG/freetz-ng/blob/master/config/avm/source.in#L460 angeschaut. Es werden die 6000 sourcen für die 6000 selbst und weiter 5 (https://github.com/Freetz-NG/freetz-ng/blob/master/config/ui/firmware.in) verwendet die einen armv8 haben, wohl weil das die erste mit sourcen war. Die neueren Sourcen hat damit niemand eingebaut/benötigt.
Kannst du dann nicht wie ein ganz normaler Mensch schreiben: "es gibt neue 4060 sourcen" ???

**Comment by SaMMy-lacht** [comment]
mit der Verwendung der richtigen source also die von der 4060 ist dieses aber nicht mehr
^

**Comment by fda77** [comment]
wie soll man die verwenden wenn die gar nicht in freetz drin sind

**Comment by SaMMy-lacht** [comment]
Das war halt nur ein Tipp, dass es vielleicht besser sein würde diese zu benutzen und nicht die vom 6000, was du damit machst, ist dir überlassen. Wenn du der Meinung bist, sie brauch nicht rein, ist das halt so.

**Comment by fda77** [comment]
Ich glaub nicht dass du meine meinung dazu wissen möchtest wie du hier wünsche äusserst

**Comment by SaMMy-lacht** [comment]
Ich habe doch kein Wunsch, denn bei mir ist es doch in freetz drin.

Es würde denn anderen aber bestimmt aber helfen, wenn du es beheben würdest ;-). Aber wie gesagt dein freetz deine Entscheidung.

**Comment by fda77** [comment]
herzlichen glückwunsch, du bist das geilste

**Reply by SaMMy-lacht** [reply]
Das war doch klar, dass es bei mir drin sein muss, sonst hätte ich das doch nicht sagen können, dass es besser sein würde es zu ändern ;-)
